src/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Flex } from '@patternfly/react-core';
+import { TrashIcon, PencilAltIcon, FileIcon } from '@patternfly/react-icons';
+
+import styles from './ActionButton.module.css';
+
+const actionIcons = {
+  create: <FileIcon />,
+  edit: <PencilAltIcon />,
+  delete: <TrashIcon />,
+};
+
+interface IActionButton {
+  actionType: 'create' | 'edit' | 'delete';
+}
+
+export const ActionButton = ({ actionType }: IActionButton) => {
```

Comment:
let's use shorter syntax in cases like this, `{ return` part can be skipped, see for example https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/components/EmptyStates/EmptyStateCard.tsx#L13